.tributors

```diff
@@ -3144,13 +3751,26 @@
         ],
         "name": "Zachary Michael"
     },
+    "zhi-de_deng": {
+        "affiliation": "National Institute of Mental Health",
+        "contributions": [
+            "doc"
+        ],
+        "email": "zhi-de.deng@nih.gov",
+        "name": "Zhi-De Deng",
+        "orcid": "0000-0001-8925-0871"
+    },
     "ezemikulan": {
         "blog": "https://github.com/ezemikulan",
         "contributions": [
             "code"
         ],
         "name": "ezemikulan"
     },
+    "jeffersoncasimir": {
+        "blog": "https://github.com/jeffersoncasimir",
+        "name": "jeffersoncasimir"
```

Comment:
@jeffersoncasimir  what is your full name (and any other info you might want to add like ORCID etc, see others) so we could acknowledge you properly?

Reply:
Thanks!

```
 "jeffersoncasimir": {
        "affiliation": "McGill Centre for Integrative Neuroscience, Montreal Neurological Institute, EEGNet, LORIS",`
        "blog": "https://github.com/jeffersoncasimir",
         "contributions": [
            "code",
            "data",
            "doc",
            "tool"
        ],
        "email": "jefferson.casimir@mcgill.ca",
        "name": "Jefferson Casimir",
        "orcid": "0009-0004-4706-0331",
        "publish_email": false
}
```

Reply:
thanks! I will redo as a separate suggestion over all lines.
note that I am not sure if `publish_email` ATM has any effect...